src/common.cu

```diff
@@ -19,6 +20,8 @@
 
 #include "../verifiable/verifiable.h"
 
+#undef RCCL_MULTIRANKPERGPU
```

Comment:
Why do we need to undef this?

Reply:
rccl team already fix this in development, and now I will remove it.